<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 6.1.1 (Python 3.12.0 on linux)" generated="20240131 16:14:44.343" rpa="false" schemaversion="4">
<suite id="s1" name="Demo" source="/workspaces/RobotFrameworkPortfolio_Salona/tests/test_suites_01/demo.robot">
<test id="s1-t1" name="Open Chrome Browser With Path To ChromeDriver" line="9">
<kw name="Open Browser" library="SeleniumLibrary">
<arg>${URL}</arg>
<arg>Chrome</arg>
<arg>executable_path=${CHROMEDRIVER_PATH}</arg>
<doc>Opens a new browser instance to the optional ``url``.</doc>
<msg timestamp="20240131 16:14:44.710" level="INFO">Opening browser 'Chrome' to base url 'https://www.google.com'.</msg>
<msg timestamp="20240131 16:14:44.716" level="INFO">Cannot capture screenshot because no browser is open.</msg>
<msg timestamp="20240131 16:14:44.716" level="FAIL">WebDriverException: Message: 'chromedriver.exe' executable may have wrong permissions.
</msg>
<status status="FAIL" starttime="20240131 16:14:44.709" endtime="20240131 16:14:44.721"/>
</kw>
<kw name="Close Browser" library="SeleniumLibrary">
<doc>Closes the current browser.</doc>
<status status="NOT RUN" starttime="20240131 16:14:44.721" endtime="20240131 16:14:44.721"/>
</kw>
<status status="FAIL" starttime="20240131 16:14:44.707" endtime="20240131 16:14:44.722">WebDriverException: Message: 'chromedriver.exe' executable may have wrong permissions.
</status>
</test>
<status status="FAIL" starttime="20240131 16:14:44.343" endtime="20240131 16:14:44.723"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" skip="0" id="s1" name="Demo">Demo</stat>
</suite>
</statistics>
<errors>
</errors>
</robot>
